HW05.Rmd

```diff
@@ -75,18 +77,16 @@ experiment_info <- read_csv("CTRP_files/experiment_info.csv")
 
 ```{r}
 
-#Bind the AUC files to get a master file
+#Bind the AUC files to get a single file
 AUC_total <- AUC_1 %>%
   bind_rows(AUC_2)
 
-glimpse(AUC_total)
-
+#Let's join with the remaining data frames
```

Comment:
Very elegant to do all of these steps together and input directly into one dataframe. Also your naming convention is clear -- it is very easy to keep track of which name corresponds to which object